extensions/binding/org.eclipse.smarthome.binding.onewire/src/main/java/org/eclipse/smarthome/binding/onewire/internal/device/OwDeviceParameter.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.binding.onewire.internal.device;
+
+import java.util.HashMap;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link OwDeviceParameter} defines a specific device map
+ *
+ * @author Jan N. Klug - Initial contribution
+ */
+public class OwDeviceParameter extends HashMap<ThingTypeUID, Object> {
```

Comment:
Its in general not a good thing to extend the Java internal collections. Are you able to refactor this so the `OwDeviceParameter` holds a map internally and delegates appropriate methods?

Reply:
done